Exercises-4-OOP/src/exercisesonclasses/Employee.java

```diff
@@ -0,0 +1,58 @@
+package exercisesonclasses;
+//Ex: The Employee Class.
+
+public class Employee {
+
+	private int ID;
```

Comment:
```suggestion
	private int id;
```
variable names should be written in campleCase format.
Make sure to fix it also in the methods and method parameters below.

Reply:
Done.